features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/sso/filter/template.jag

```diff
@@ -37,6 +37,7 @@
 	if (checkSSO && !Boolean(isAuthenticated)) {
 
 	    var keyStorePassword = site.ssoConfiguration.keyStorePassword;
+        var acsURL = site.ssoConfiguration.acsURL;

```

Comment:
Fixed in #1534 (https://github.com/wso2/carbon-apimgt/commit/39eedc9c22f37a64d82698b95f95a166f9c53a33)